udfs/js_libs/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "js-bq-libs",
+  "version": "1.0.0",
+  "scripts": {
+    "build-all-libs": "concurrently \"npm:webpack-*\"",
+    "webpack-compromise-v11.14.0": "webpack --config compromise-v11.14.0-webpack.config.js",
+    "webpack-js-levenshtein-v1.1.6": "webpack --config js-levenshtein-v1.1.6-webpack.config.js",
```

Comment:
this PR should also update the UDF Definition queries to point to these "managed" GCS URIs.

Reply:
oops looks like it already does! sorry I missed it.